ext/ffi/ffi.c

```diff
@@ -4469,6 +4469,318 @@ ZEND_METHOD(FFI_CType, getName) /* {{{ */
 		RETURN_STR(res);
 	}
 }
+/* }}} */
+
+ZEND_METHOD(FFI_CType, getKind) /* {{{ */
+{
+	zend_ffi_ctype *ctype = (zend_ffi_ctype*)(Z_OBJ_P(ZEND_THIS));
+	zend_ffi_type *type;
+
+	if (zend_parse_parameters_none() == FAILURE) {
+		RETURN_THROWS();
+	}
+
+	type = ZEND_FFI_TYPE(ctype->type);
+	RETURN_LONG(type->kind);
+}
+/* }}} */
+
+ZEND_METHOD(FFI_CType, getSize) /* {{{ */
+{
+	zend_ffi_ctype *ctype = (zend_ffi_ctype*)(Z_OBJ_P(ZEND_THIS));
+	zend_ffi_type *type;
+
+	if (zend_parse_parameters_none() == FAILURE) {
+		RETURN_THROWS();
+	}
+
+	type = ZEND_FFI_TYPE(ctype->type);
+	RETURN_LONG(type->size);
+}
+/* }}} */
+
+ZEND_METHOD(FFI_CType, getAlignment) /* {{{ */
+{
+	zend_ffi_ctype *ctype = (zend_ffi_ctype*)(Z_OBJ_P(ZEND_THIS));
+	zend_ffi_type *type;
+
+	if (zend_parse_parameters_none() == FAILURE) {
+		RETURN_THROWS();
+	}
+
+	type = ZEND_FFI_TYPE(ctype->type);
+	RETURN_LONG(type->align);
+}
+/* }}} */
+
+ZEND_METHOD(FFI_CType, getAttributes) /* {{{ */
+{
+	zend_ffi_ctype *ctype = (zend_ffi_ctype*)(Z_OBJ_P(ZEND_THIS));
+	zend_ffi_type *type;
+
+	if (zend_parse_parameters_none() == FAILURE) {
+		RETURN_THROWS();
+	}
+
+	type = ZEND_FFI_TYPE(ctype->type);
+	RETURN_LONG(type->attr);
+}
+/* }}} */
+
+ZEND_METHOD(FFI_CType, getEnumKind) /* {{{ */
+{
+	zend_ffi_ctype *ctype = (zend_ffi_ctype*)(Z_OBJ_P(ZEND_THIS));
+	zend_ffi_type *type;
+
+	if (zend_parse_parameters_none() == FAILURE) {
+		RETURN_THROWS();
+	}
```

Comment:
```suggestion
    ZEND_PARSE_PARAMETERS_NONE();
```

nit: Could be shortened here and elsewhere. Is there a reason for different parts of the codebase using different ways of writing this?